exporter/opentelemetry-exporter-zipkin/src/opentelemetry/exporter/zipkin/__init__.py

```diff
@@ -364,6 +371,45 @@ def _extract_tags_from_dict(self, tags_dict):
             tags[attribute_key] = value
         return tags
 
+    def _extract_tag_value_string_from_sequence(self, sequence):
+        if not sequence:
+            return None
+
+        if self.max_tag_value_length == 1:
+            return None
+
+        tag_value_elements = []
+        running_string_length = (
+            2  # accounts for array brackets in output string
+        )
+        defined_max_tag_value_length = self.max_tag_value_length > 0
+
+        if isinstance(sequence, (list, tuple, range)):
+            for element in sequence:
+                if isinstance(element, (int, bool, float)):
+                    tag_value_element = str(element)
+                elif isinstance(element, str):
+                    tag_value_element = element
+                else:
+                    return None
+
+                if defined_max_tag_value_length:
+                    running_string_length += (
+                        len(tag_value_element) + 2
+                    )  # +2 accounts for surrounding quotes
+                    if tag_value_elements:
+                        running_string_length += (
+                            2  # accounts for ', ' connector
+                        )
+                    if running_string_length > self.max_tag_value_length:
```

Comment:
Is `max_tag_value_length` supposed to be encoded byte length or string length?

Reply:
We're outputting as ASCII with json.dumps() so one and the same.

Reply:
I'm not sure actually so I checked the python docs:

> If ensure_ascii is true (the default), the output is guaranteed to have all incoming non-ASCII characters escaped.

So I think if the escape sequences are long, you could get longer length. I think this is OK tho 😄 Maybe just update `max_tag_value_length` docstring to say its string length if not specified?